ui-playground/manifest.json

```diff
@@ -0,0 +1,73 @@
+{
+  "manifest_version": 2,
+  "icons": {
+    "16": "images/whimsycorn-18.png",
+    "19": "images/whimsycorn-48.png",
+    "32": "images/whimsycorn-32.png",
+    "38": "images/whimsycorn-128.png",
+    "48": "images/whimsycorn-48.png",
+    "64": "images/whimsycorn-64.png",
+    "128": "images/whimsycorn-128.png"
+  },
+  "applications": {
+    "gecko": {
+      "id": "ui-playground@mozilla",
+      "strict_min_version": "40.0.0"
+    }
+  },
+  "permissions": [
+    "tabs",
```

Comment:
I was surprised you needed "tabs" for this. It seems like in Firefox you need it in order to query by URL, or you get a failure here: https://dxr.mozilla.org/mozilla-central/source/browser/components/extensions/ext-tabs.js#431 because tab.url is undefined. But in Chrome you don't. I'll file a bug for this.